chef/data_bags/crowbar/bc-template-network.schema

```diff
@@ -82,6 +82,8 @@
                     "vlan": { "type": "int", "required": true },
                     "use_vlan": { "type": "bool", "required": true },
                     "add_bridge": { "type": "bool", "required": true },
+                    "add_ovs_bridge": { "type": "bool", "required": false },
+                    "bridge_name": { "type": "str", "required": false },
```

Comment:
Note: I defined those to attribute to be optional because they should only ever be needed for the "nova_floating" and "nova_fixed" networks (and any addtional external network that might get added). I'd like to avoid them being added to all the other networks ("admin", "os_sdn", ...). IMO that would just cause confusion.